apps/rush-lib/.npmignore

```diff
@@ -21,4 +21,5 @@
 ## Project specific definitions
 # -----------------------------
 
-# (Add your exceptions here)
\ No newline at end of file
+# (Add your exceptions here)
+!/etc/**
```

Comment:
restrict this to examples

Reply:
done

---
In reply to: [148912766](https://github.com/Microsoft/web-build-tools/pull/431#discussion_r148912766) [](ancestors = 148912766)